zappa/core.py

```diff
@@ -2098,9 +2098,21 @@ def get_rest_apis(self, project_name):
         """
         Generator that allows to iterate per every available apis.
         """
-        all_apis = self.apigateway_client.get_rest_apis(limit=500)
+        all_apis = []
 
-        for api in all_apis["items"]:
+        pagination_position = "init"
```

Comment:
Just thinking since it's yielding the result there's no need to collect all results into a list then yield.
How about yielding results on each response?

Something like:
```
response = self.apigateway_client.get_rest_apis(limit=500)
yield from (api for api in response["items"] if api["name"] == project_name)
while response.get("position"):
    response = self.apigateway_client.get_rest_apis(limit=500, position=response["position"])
    yield from (api for api in response["items"] if api["name"] == project_name)
```

Reply:
@DilLip-Chowdary-Codes Do you have time to review/update or is it better for one of the core developers to take this over?